rero_ils/modules/loans/api.py

```diff
@@ -165,6 +165,20 @@ def library_pid(self):
             return loc.library_pid
         return None
 
+    @property
+    def location_pid(self):
+        """Get location PID regarding transaction Location PID or location."""
+        from ..items.api import Item
+
+        location_pid = self.get('transaction_location_pid')
```

Comment:
can be rewritten as:
```python
location_pid = self.get('transaction_location_pid')
if location_pid:
    return location_pid
item_pid = self.get('item_pid')
if item_pid:
    return Item.get_record_by_pid(item_pid).location_pid
# optional
return None
```

Reply:
You're right. I based on `def library_pid` method directly above. I change the code of this method too.